src/utils/test-helpers.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Taken from https://github.com/milesj/boost/blob/master/packages/cli/tests/helpers.ts
+ */
+export const KEYS = {
+    // backspace: isWindows ? '\u0008' : '\u007F',
+    // delete: isWindows ? '\u007F' : '\u001B[3~',
+    backspace: '\u0008',
+    delete: '\u007F',
```

Comment:
Should we delete the comments?

Reply:
I was scared to tbh, the guy who I took this from seemed concerned about windows support and so am I, wanted to have these handy at least in commit history (since I'll squash this commit) - down to delete after though if it feels gross